zeebe/exporters/opensearch-exporter/src/test/java/io/camunda/zeebe/exporter/opensearch/TestSupport.java

```diff
@@ -120,7 +120,9 @@ static Stream<ValueType> provideValueTypes() {
             ValueType.SBE_UNKNOWN,
             ValueType.NULL_VAL,
             ValueType.PROCESS_INSTANCE_RESULT,
-            ValueType.CLOCK);
+            ValueType.CLOCK,
+            // these are not yet supported
+            ValueType.ROLE);
```

Comment:
Adding this here as I'm not sure if following the old approach is still valid

Reply:
Same comment as vor the ES exporter